package aj.dev.event.view.checkin.exception

internal class ValidateNameException(msg: String) : RuntimeException(msg)